client/src/components/AutoSleepToggle/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react'
+import NoSleep from 'nosleep.js'
+import { isMobile } from '@/utils'
+import { Toggle } from "@/components";
+import './style.scss'
+
+const AutoSleepToggle = ({ noSleepEnabled, setNoSleepEnabled }: { noSleepEnabled: boolean, setNoSleepEnabled: (arg0: boolean) => void }): JSX.Element => {
+  const noSleep = new NoSleep()
+
+  useEffect(() => {
+    if (isMobile) {
+      noSleepEnabled ? noSleep.enable() : noSleep.disable()
+    }
+  }, [noSleepEnabled])
+
+  return isMobile ? (
```

Comment:
The check for isMobile might need to be in the RoomPage since rn, the divider shows up even on the desktop which looks weird.
![image](https://user-images.githubusercontent.com/34047281/202063050-f80f6a2e-a06b-4b54-853c-41f0ed0057cb.png)